web/scss/pages/_ancillary.scss

```diff
@@ -0,0 +1,219 @@
+.ancillary {
+  blockquote,
+  iframe,
+  ol,
+  p,
+  ul {
+    margin-bottom: 1.25rem;
+  }
+
+  ol,
+  p,
```

Comment:
Do we need unique content stylings for our ancillary pages vs. blog posts or other more mainstream pages? Same for tables and probably some other rules in this sheet.

Reply:
Everything in this file will most likely be moved to the "rich text" component. I gotta look at how the app is doing things and basically pick and choose. Or, just finish this site and make the app use the same thing.